src/components/shared/Entity/Actions/SaveButton.tsx

```diff
@@ -0,0 +1,79 @@
+import { EditorStoreState } from 'components/editor/Store';
+import EntityCreateSave from 'components/shared/Entity/Actions/Save';
+import LogDialog from 'components/shared/Entity/LogDialog';
+import LogDialogActions from 'components/shared/Entity/LogDialogActions';
+import { DraftSpecQuery } from 'hooks/useDraftSpecs';
+import { useRouteStore } from 'hooks/useRouteStore';
+import { useZustandStore } from 'hooks/useZustand';
+import { FormattedMessage } from 'react-intl';
+import { entityCreateStoreSelectors, FormStatus } from 'stores/Create';
+
+interface Props {
+    closeLogs: Function;
+    callFailed: Function;
+    disabled: boolean;
+    materialize?: Function;
+}
+
+function EntitySaveButton({
```

Comment:
This component is to house the save button with it's own dialog.  Previous `savebutton` is now `save`